README.md

```diff
@@ -106,6 +106,9 @@ pauses between various steps so that you can follow what the test is doing more
 The e2e test for find currently requires a `test`-scoped API key for GOV.UK Notify to retrieve the email send during
 the test.  Pass an environment variable called `E2E_NOTIFY_FIND_API_KEY` to allow this test to pass.
 
+If you want to run the end-to-end tests against our deployed dev/test environments, you can do this by adding the
+`--e2e-env` flag to the pytest command with a value of either `dev` or `test`.
```

Comment:
I was prompted to run `playwright install` do we need a pre-requisite about this?
```
           playwright._impl._errors.Error: BrowserType.launch: Executable doesn't exist at /Users/gideongoldberg/Library/Caches/ms-playwright/chromium-1124/chrome-mac/Chromium.app/Contents/MacOS/Chromium
E           ╔════════════════════════════════════════════════════════════╗
E           ║ Looks like Playwright was just installed or updated.       ║
E           ║ Please run the following command to download new browsers: ║
E           ║                                                            ║
E           ║     playwright install                                     ║
E           ║                                                            ║
E           ║ <3 Playwright Team                                         ║
E           ╚════════════════════════════════════════════════════════════╝
```

Reply:
It's documented in the readme a few lines above.

Reply:
fair enough!